Drop/Drop/CenterAlignedCollectionViewFlowLayout.swift

```diff
@@ -0,0 +1,84 @@
+//
```

Comment:
Yeah, it was a failed attempt on aligning players in the lobby centered horizontally (: